app/controllers/profile_controller.rb

```diff
@@ -45,8 +46,10 @@ def cabinet
     @projects = current_user.projects if current_user
 
     if @user.developer?
+      @approved = Project.includes(:user).where(employee: current_user.id)
       render 'profile/developer_cabinet'
     else
+      @implementation_project = true if current_user.projects.where(status: 1).length.positive?
```

Comment:
current_user.projects.notification_sent.length.positive? instead of current_user.projects.where(status: 1).length.positive?